rpc/src/test/scala/org/bitcoins/rpc/v17/BitcoindV17RpcClientTest.scala

```diff
@@ -131,7 +134,7 @@ class BitcoindV17RpcClientTest extends AsyncFlatSpec with BeforeAndAfterAll {
     val l = "setLabel"
     val btc = Bitcoins(1)
     val addr = await(client.getNewAddress)
-    await(TestUtil.fundBlockChainTransaction(otherClient, addr, btc))
+    await(BitcoindRpcTestUtil.fundBlockChainTransaction(otherClient, addr, btc))
```

Comment:
I think it is better to do something like 

```scala
val _ = await(BitcoindRpcTestUtil.fundBlockChainTransaction(otherClient,addr,btc))
```

This is says to the reader of this code that `fundBlockchainTransaction` returns something, but we do not care about it in this context. Thus the `val _ = ...`